apps/emqx_ds_builtin_raft/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -956,6 +966,19 @@ forget_node(Node) ->
             logger:error("Failed to forget leaving node ~p: ~p", [Node, Reason])
     end.
 
+forget_lost_sites() ->
+    NodesLost = [R || R = #?NODE_TAB{node = N} <- all_nodes(), node_status(N) =:= lost],
```

Comment:
Tiny nit:

```suggestion
    NodesLost = sites(lost),
```

Reply:
Yeah but the thing is we need whole records later, not just `site()`s. 🫠